cmd/config.go

```diff
@@ -66,7 +67,12 @@ func createConfigPath() error {
 	if err != nil {
 		return err
 	}
-	return os.MkdirAll(configPath, 0700)
+	err = os.MkdirAll(configPath, 0700)
+	if err != nil {
+		return err
+	}
+	console.Println("Created minio config dir at " + configPath)
```

Comment:
This should happen after "createConfigPath()" has returned. Print this in "initConfig()"